functions/src/definitions/webhookHandler.ts

```diff
@@ -70,16 +61,19 @@ app.post("/whatsapp", async (req, res) => {
             (type == "button" || type == "interactive" || type == "text") &&
             phoneNumberId === checkerPhoneNumberId
           ) {
-            //when live, can check against WABA id instead
-            await checkerHandlerWhatsapp(message)
+            //put into checker queue
```

Comment:
Not a queue. pubsub messages are consumed once the subber touches it, and not when a success is returned.

Reply:
Isit? I think it consumes on success only leh, see here:

https://cloud.google.com/pubsub/docs/push

> Here's a brief description of the workflow that references Figure 3:
> 
> The Pub/Sub server sends each message as an HTTPS request to the subscriber client at a pre-configured endpoint. This request is shown as a PushRequest in the image.
> The endpoint acknowledges the message by returning an HTTP success status code. A non-success response indicates that Pub/Sub must resend the messages. This response is shown as a PushResponse in the image.
> Pub/Sub dynamically adjusts the rate of push requests based on the rate at which it receives success responses.